cnxarchive/views.py

```diff
@@ -981,3 +1016,47 @@ def robots(request):
     resp.expires = html_date(exp_time)
     resp.body = robots_dot_txt.to_string()
     return resp
+
+@view_config(route_name='recent', request_method='GET',
+             accept="application/json",
+             renderer='templates/recent.rss') #renderer: cnxarchive.views:templates/recent.html?? also what should the permission actuall be??
```

Comment:
Let's remove the comment here. Also, let's remove the `accept` argument, it's not useful in this case.